cmd/zc_traverser_blob.go

```diff
@@ -377,10 +377,13 @@ func (t *blobTraverser) createStoredObjectForBlob(preprocessor objectMorpher, bl
 		containerName,
 	)
 
-	object.blobSnapshotID = blobInfo.Snapshot
 	object.blobDeleted = blobInfo.Deleted
-	if blobInfo.VersionID != nil {
-		object.blobVersionID = *blobInfo.VersionID
+	if t.includeDeleted && t.includeSnapshot {
+		object.blobSnapshotID = blobInfo.Snapshot
+	} else if t.includeDeleted && t.includeVersion {
+		if blobInfo.VersionID != nil {
```

Comment:
Nit: Can be included in the above else condition. Is this a regression?

Reply:
Yeah, it wasn't picked up by the CI unfortunately.